integration/src/test/java/edu/unc/lib/boxc/integration/web/access/SearchEndpointIT.java

```diff
@@ -348,16 +347,58 @@ public void testSearchWithCreatedDateRangeSpecifiedAsUnknown() throws Exception
         }
     }
 
-    private List<String> createDatedObjects() throws Exception {
-        List<String> ids = new ArrayList<>();
-        var datedAdminUnit = adminUnitFactory.createAdminUnit(
-                Map.of("title", "Dated Admin Object", "dateCreated", "2018-07-01"));
-        var datedCollection = collectionFactory.createCollection(datedAdminUnit,
-                Map.of("title", "A dated collection",
-                        "dateCreated", "2022-07-01",
-                        "readGroup", "everyone"));
-        ids.add(datedAdminUnit.getPid().getId());
-        ids.add(datedCollection.getPid().getId());
-        return ids;
+    @Test
+    public void testSearchWithOneFacetFilter() throws Exception {
+        createDefaultObjects();
+        createLanguageSubjectObjects();
+
+        var getMethod = new HttpGet(SEARCH_URL + "/?facetSelect=language&language=English&getFacets=true");
+
+        try (var resp = httpClient.execute(getMethod)) {
+            var facetFields = getNodeFromResponse(resp, "facetFields");
+
+            assertSuccessfulResponse(resp);
+            // should only find the language facet
+            assertEquals(1, facetFields.size());
+        }
+    }
+
+    @Test
+    public void testSearchWithLanguageFilterSelectedDisplaysAllAvailableLanguages() throws Exception {
+        createDefaultObjects();
+        createLanguageSubjectObjects();
+
+        var getMethod = new HttpGet(SEARCH_URL + "/?facetSelect=language&language=English&getFacets=true");
+
+        try (var resp = httpClient.execute(getMethod)) {
+            var facetFields = getNodeFromResponse(resp, "facetFields");
+            var languageFields = new ArrayList<JsonNode>();
+            facetFields.get(0).get("values").elements().forEachRemaining(languageFields::add);
+
+            assertSuccessfulResponse(resp);
+            // should find all available languages: English, Cherokee
+            assertValuePresent(languageFields,0,"value", "English");
+            assertValuePresent(languageFields,1,"value", "Cherokee");
+        }
+    }
+
+    @Test
+    public void testSearchWithSubjectFilterSelectedDisplaysAvailableLanguages() throws Exception {
+        createDefaultObjects();
+        createLanguageSubjectObjects();
+
+        var getMethod = new HttpGet(SEARCH_URL + "/?facetSelect=language%2Csubject&subject=North%2520Carolina&getFacets=true");
+
+        try (var resp = httpClient.execute(getMethod)) {
+            var facetFields = getNodeFromResponse(resp, "facetFields");
+            var languageFields = new ArrayList<JsonNode>();
+            facetFields.get(0).get("values").elements().forEachRemaining(languageFields::add);
+
+            assertSuccessfulResponse(resp);
+            // should find all available languages: English
+            assertEquals(2,facetFields.size());
```

Comment:
Since this is the only test that is verifying two facets being selected, it'd be good to add an assertion that the subject field is also being returned.